streams/sv.m3u

```diff
@@ -1,8 +1,6 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="Canal10.sv" status="blocked",Canal 10 (720p)
 http://streamingcws20.com:1935/tves/tves.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="Canal12.sv" status="online",Canal 12
-https://d3kk75blondskv.cloudfront.net/hls/canal12.m3u8
```

Comment:
Wasn't this supposed to be replaced by 
https://do0b1lvn4z8ni.cloudfront.net/hls/canal12.m3u8